app/models/user_like_tweet.rb

```diff
@@ -0,0 +1,7 @@
+class UserLikeTweet < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :tweet
+
+  validates :user,  presence: true
+  validates :tweet,  presence: true
```

Comment:
You should add a validation to check the records uniqueness. A record with the same user and tweet shouldn't be repeated on the database. You can use the uniqueness validator with a scope (http://guides.rubyonrails.org/active_record_validations.html#uniqueness) to solve this